CRM/Pledge/BAO/PledgeBlock.php

```diff
@@ -38,8 +38,8 @@ public static function retrieve($params, &$defaults) {
    *
    * @param array $params
    *   (reference ) an assoc array of name/value pairs.
-   *
-   * @return CRM_Pledge_BAO_PledgeBlock
+   * @deprecated
+   * @return CRM_Pledge_DAO_PledgeBlock
    */
```

Comment:
OK - this is long but completely lacking in goodness 
![image](https://user-images.githubusercontent.com/336308/180616291-2d5d09e7-5165-4498-a013-ad1d50664aac.png)